### PR TITLE
fix(MapBox): Upgrade to latest MapBox API

### PR DESCRIPTION
**Pull request by GordonSmith**
Previous API expires June 1, 2020

Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage please review

**Comment by GordonSmith** [comment]
@jbrundage please review

**Comment by GordonSmith** [comment]
@jbrundage bump

File: packages/map/src/leaflet/MapBox.ts [hunk 1]
```diff
@@ -24,10 +24,10 @@ export class MapBoxLayer extends TileLayer {
             console.warn("__hpcc_mapbox_apikey does not contain a valid API key, reverting to developers key (expect limited performance)");
         }
         const hpcc_mapbox_apikey = window.__hpcc_mapbox_apikey || "pk.eyJ1IjoibGVzY2htb28iLCJhIjoiY2psY2FqY3l3MDhqNDN3cDl1MzFmZnkwcCJ9.HRoFwmz1j80gyz18ruggqw";
-        this.add(new LeafletTileLayer(`https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token=${hpcc_mapbox_apikey}`, {
+        this.add(new LeafletTileLayer(`https://api.mapbox.com/styles/v1/mapbox/{id}/tiles/{z}/{x}/{y}?access_token=${hpcc_mapbox_apikey}`, {
             attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
             maxZoom: 18,
-            id: "mapbox.streets",
+            id: "streets-v11",
```
Comment (jbrundage) [inline packages/map/src/leaflet/MapBox.ts:30]: Cool beans... their styles API looks great

Comment (jbrundage) [reply packages/map/src/leaflet/MapBox.ts:30]: Can we simply swap line 30 out for one of these to get some variance?
- outdoors-v11
- light-v10
- dark-v10
- satellite-v9
- satellite-streets-v11

Comment (GordonSmith) [reply packages/map/src/leaflet/MapBox.ts:30]: Yes

**Pull request merged**